### DISCUSSION TITLE
How to get the Pull Shark Badge?

### DISCUSSION THREAD

**Discussion by kurodahiroyuki23** [community/community#33542] [Other Feature Feedback, Questions, & Ideas] [answered]
I would like to get the pull shark badge on my github profile.

**Comment by strong-728** [comment, marked as answer]
Once you make 2 merged pull request, you will get the one pull shark badge.
the 2x is 16th merged pull request.
Thanks.

**Reply by xmas7** [reply]
PullShark x 1 => 2 Pull Requets (PRs)
PullShark x 2 => 16
PullShark x 3 => 128
PullShark x 4 => 1024

**Reply by sathlerhe** [reply]
I have to merge or open the pr?

**Reply by Vellutia** [reply]
@sathlerhe you have to be the one who open the PR

**Reply by strong-728** [reply]
only need to open the PR.

**Reply by sandeepnmenon** [reply]
I see under my PR tab that I have closed 129 PRs. Still I haven't gotten the x3 badge. Is it number of PRs in a year?
<img width="665" alt="image" src="https://user-images.githubusercontent.com/11742991/234669953-9e9eeee3-d005-45e3-a365-14615823d028.png">

**Reply by chmodsss** [reply]
The PR has to be merged. closed PRs also mean, some could be simply rejected without merging.

**Reply by supul-k** [reply]
It should be merged pull requests not closed pull requests

**Reply by kaangulbahar** [reply]
@supul-k Thank you tho, this helped.

**Reply by mayachee** [reply]
You can get the pull shark badge by having your PRs merged into another repo!

**Reply by LucasAlmeida-jpg** [reply]
guys, I got more than 16  merged pull requests, and until now it does not appear 2x, is it normal?

**Reply by samueltuoyo15** [reply]
> guys, I got more than 16 merged pull requests, and until now it does not appear 2x, is it normal?

Same as me I got 17th pull request that was merged and my shark badged doesn't still appears as X2 why ??

This is my Profile:
https://github.com/samueltuoyo15


![Screenshot_20241223-140538](https://github.com/user-attachments/assets/d669a93e-550d-4aa0-98b0-9e164f350bc2)

**Reply by LucasAlmeida-jpg** [reply]
> guys, I got more than 16  merged pull requests, and until now it does not appear 2x, is it normal?

Now it is showing up the badget 2x, cause all of the 16 PR has a little checked icon in front of the PR. Pay attention in this little observation.

**Reply by samueltuoyo15** [reply]
It's not now I have over 19 I still don't have the pull Shark X2 😭💔

**Reply by muthupriyan-dev** [reply]
Hello friends

My GitHub username is: muthupriyan-dev.

I have multiple merged pull requests in my public repository, but I still haven't received the Pull Shark achievement.

Repository:
https://github.com/muthupriyan-dev/tic-tac-toe

Could you please check if my merged pull requests are eligible for the Pull Shark achievement, or if there is any issue with my account?

Thank you.

**Reply by v-ember** [reply]
Hi @muthupriyan-dev ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).r

**Comment by ezimoha-obichukwu** [comment]
Thanks

**Comment by bumpedyou** [comment]
Hi, Software provider
I have some question for you. How to get air pollution on Google Map ? 
Thanks

**Reply by softawre-developer215** [reply]
you can use https://console.cloud.google.com/

**Reply by bumpedyou** [reply]
Let me check

**Comment by corestack-cs** [comment]
need to check settings on google map api

**Reply by bumpedyou** [reply]
Thanks

**Comment by bumpedyou** [comment]
Thanks jamesVectorspace

**Comment by asifsamir** [comment]
Thanks. 👍

**Comment by khalid586** [comment]
Get 2 pull requests merged. Then you will get a pull shark badge.

**Reply by sahilwadaskar07** [reply]
I have merged 3 PR's but cant get Shark badge still now.

**Comment by softawre-developer215** [comment]
How can i get Galaxy Brain and Pair Extraordinaire badge?

**Reply by h7x4** [reply]
Here's an overview of the different badges: https://github.com/drknzz/GitHub-Achievements

#### Galaxy Brain:

Answer a discussion post, just like this, and get your answer marked as the accepted answer.

#### Pair Extraordinaire:

When you collaborate with someone on a commit, they might declare so in the bottom of the commit message by using a special [`Co-authored-by: name <name@email.com>` syntax](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors). GitHub will recognize this, and mark the commit with 2 (or more) authors. If such a commit is included in a PR and it gets merged, you're on your way to get this badge.

**Reply by bumpedyou** [reply]
I can help you

**Reply by Mannyflow7** [reply]
#@[Select]([url](url))[
[Submit feedbackHello . Please make sure to address me by my name, , when responding to messages or emails as sender. Thank ###  @you.](url) ](url)

**Comment by TRZMlEL** [comment]
You can get this badge by merging 2 pull requests.

**Comment by Mannyflow7** [comment]
> I can help you
[
](https://github.com/orgs/community/discussions/33542#discussioncomment-6621330)

**Reply by softawre-developer215** [reply]
how can i get Galaxy Brain badge?

**Comment by Mannyflow7** [comment]
> I would like to get the pull shark badge on my github profile.
[
](https://github.com/orgs/community/discussions/33542#discussion-4405354[WORK REGISTRATION INSTRUCTIONS.docx](https://github.com/community/community/files/12245239/WORK.REGISTRATION.INSTRUCTIONS.docx)

**Comment by hammadalive7** [comment]
For Getting  pull shark badge you have to create 2 pull requests to any repo and make sure they get merged.
You can find more about badges here: https://github.com/Schweinepriester/github-profile-achievements

**Comment by aliasgharfathikhah** [comment]
Hello
In order to receive this badge, you need to have two merged **pull request**, which you have entered into a repository for someone else or you have merged for yourself.

Sometimes there is a problem and it is not shown to you, like me when I had 8 **pull request**, then I told the support and they fixed it for me.

**Reply by Adesoji1** [reply]
@aliasgharfathikhah how did you contact support

**Reply by aliasgharfathikhah** [reply]
I contacted support via this email:
**_developer@githubsupport.com_**
And maybe this link will help you:
**_https://support.github.com/_**

**Comment by navodlakshan** [comment]
Open two pull requests and merge them. Then you can get the Pull Shark achievement badge. Follow the link below for more information.
Link: https://github.com/drknzz/GitHub-Achievements?tab=readme-ov-file

**Comment by CryptoNinja0331** [comment]
thanks.

**Comment by ColdFire69** [comment]
How about the OTHER badges?

**Comment by samueltuoyo15** [comment]
Thanks for your feedback but you haven't resolve the issue

I already have the shark badge but it's not showing X2 Even though I've
obtained 19 merged pull request

Still yet it's not showing X2

Please kindly examine my account
username: samueltuoyo15

Attached to this letter is a copy of screenshots of my issue 🤧

I highly anticipate your reply

**Reply by LucasAlmeida-jpg** [reply]
> username
Hey buddy, is all of your PR is approved and has a little checked icon in front of them? If not, it is not gonna show up.

**Comment by samueltuoyo15** [comment]
How can I check if the PR has a green check icon

Cuz 19 of my PR was merged successfully

Is there anyway for you to resolve this issue from your end?

**Reply by LucasAlmeida-jpg** [reply]
> How can I check if the PR has a green check icon
> 
> 
> 
> Cuz 19 of my PR was merged successfully
> 
> 
> 
> Is there anyway for you to resolve this issue from your end?
> 
> 
> 
> On Mon, Dec 30, 2024, 8:41 PM Lucas Almeida ***@***.***>
> 
> wrote:
> 
> 
> 
> > username
> 
> > Hey buddy, is all of your PR is approved and has a little checked icon in
> 
> > front of them? If not, it is not gonna show up.
> 
> >
> 
> > —
> 
> > Reply to this email directly, view it on GitHub
> 
> > <https://github.com/orgs/community/discussions/33542#discussioncomment-11699390>,
> 
> > or unsubscribe
> 
> > <https://github.com/notifications/unsubscribe-auth/BLV373HLRDSLPOPQOTCWHN32IGOWLAVCNFSM6AAAAAAQQSVTAGVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNRZHEZTSMA>
> 
> > .
> 
> > You are receiving this because you commented.Message ID:
> 
> > ***@***.***>
> 
> >
> 
> 

![image](https://github.com/user-attachments/assets/b6176255-ae32-474b-beec-ee7e4f5758d2)

No, I am just an user as you. 
You got to take a look in every approved PR that you got, and check to ser if there is a icon like this one attached.

**Comment by TahaAyati-DevLab** [comment]
Once you make 2 merged pull request, you will get the one pull shark badge.
the 2x is 16th merged pull request.
Thanks.
